bin/fin

```diff
@@ -1313,11 +1373,14 @@ _start_containers ()
 	check_docker_path
 
 	echo-green "Starting services..."
-	docker-compose up -d --remove-orphans --build || return 1
+	docker-compose up -d --remove-orphans --build --quiet-pull || return 1
```

Comment:
Pull can take many minutes when install is fresh. Progress was providing valuable visual feedback. --quiet-pull will remove that

Reply:
While it looks nice in an interactive shell, pull produces a log of garbage in the CI logs.

Interactive shell:

<img width="743" alt="Screen Shot 2021-11-09 at 08 27 26" src="https://user-images.githubusercontent.com/1205005/140964845-8c7ec397-02b1-4f9a-a532-a1410f0c33a6.png">

Ci logs:

<img width="767" alt="Screen Shot 2021-11-09 at 08 29 02" src="https://user-images.githubusercontent.com/1205005/140964703-5e710cba-4d41-4ee5-856e-0cfb4e652b45.png">

With `--quiet-pull` user still gets a "spinner" instead of the full progress output:

<img width="167" alt="Screen Shot 2021-11-09 at 08 26 23" src="https://user-images.githubusercontent.com/1205005/140965113-b81c7b97-9976-4007-97b8-0088cc555e86.png">

@achekulaev  If you feel like the humble pull spinner output is not sufficient, then we can do `--quiet-pull` only for non-interactive shells (`is_tty != 0`).

Note: CIs that fake TTY (I know Travis does this, not sure about the rest) would still suffer from the enormous pull output.

Reply:
I am a fan of seeing the download piece to see the true progression of everything.

Reply:
Switched to doing `--quiet-pull` only for CI environments.